### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by CPMeade**
This PR will add in the Profile page as well as making many of the links to the profile page work properly

### CODE REVIEW

**Comment by CPMeade** [comment]
> This looks good, but there's a big blank spot in the upper-left hand corner and I'm wondering if it's possible for a user to add a profile picture?

Currently, there is not a spot in the code for a profile picture, because there is not a picture for profiles in Strapi.

It would not be too difficult to add in a section for the picture however, since the code for the profile page is structured based off of the project page, and a picture could be added in the same part of the code.

**Comment by liliedelamotte** [comment]
I see. Would have been nice to add a profile picture but since we're out of time, not sure we can make it happen. Not sure we should push this, then, if it's slightly unfinished? Anyone else have thoughts on this?

**Comment by kjh61998** [comment]
> I see. Would have been nice to add a profile picture but since we're out of time, not sure we can make it happen. Not sure we should push this, then, if it's slightly unfinished? Anyone else have thoughts on this?

We're going to add more data, pull from this branch, and try to get it to have a more finished look. I think that either it shouldn't get merged until it's a little more finished.

**Comment by kjh61998** [comment]
It's not perfect, but I added an image and tried to do something more interesting with the data. Let me know what you guys think!

File: src/components/ProjectTile.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const ProjectTile = ({ data }) => (
       <Link style={{display:'inline-block', color: '#1C2833'}} to={`/${data.id}`}>{data.project_name}</Link>
     </h3>
     <h4 className="has-text-weight-normal is-size-5 is-size-6" style={{color: '#1C2833'}}>
-      <Link style={{color: '#1C2833'}} to={`/${data.profiles.profile_name}`}><i>Sponsored by {data.profiles.profile_name}</i></Link>
+      <Link style={{color: '#1C2833'}} to={`/Profile_${data.profiles.id}`}><i>Sponsored by {data.profiles.profile_name}</i></Link>
```
Comment (chris-hammond) [inline src/components/ProjectTile.js]: I believe this should be `has-text-primary` or something similar?

File: src/components/PreviewCompatibleImage.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import Img from 'gatsby-image'
 
 const PreviewCompatibleImage = ({ imageInfo }) => {
-  const imageStyle = { borderRadius: '5px' }
+  const imageStyle = { borderRadius: '5px' , maxHeight:'800px', maxWidth:'800px'}
```
Comment (chris-hammond) [inline src/components/PreviewCompatibleImage.js]: Why is this here? It's nit used on the profile page, so can we just revert this line? I prefer to use the image processing graphql queries to set image sizes, as it ensures we aren't accidentally using a 3MB image file for a thumbnail...

File: src/templates/specific-profile.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                        <div>
+                            {experience.experience_description}
+                        </div>
+                    </div>
+                ))}
+              </div> <br /><br />
+          </div>
+
+
+          <div label="Endorsements" icon="FaList">
```
Comment (chris-hammond) [inline src/templates/specific-profile.js]: What's the plan with Endorsements? Should we hide them or actually add them in strapi? Or are we fine with this approach?

File: src/components/all.sass [hunk 4]
```diff
@@ -53,5 +53,26 @@ $body-color: $tech-connect-blue
   background-color: $tech-connect-blue
   height: 4px
 
+.tab-list
+  border-bottom: 1px solid #ccc
```
Comment (chris-hammond) [inline src/components/all.sass]: Should #ccc be given a variable name at the top of the file?

File: src/templates/specific-profile.js [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            <br /><br />
+          </div>
+
+          <div label="Projects" icon="FaPagelines">
+              <div className="columns is-8 is-multiline is-centered">
+                {data.strapiProfile.projects.map(project => (
+                    <div className="column is-8- mobile is-10-tablet is-4-desktop is-4-widescreen">
+                    <Link to={`/Project_${project.id}`}><PreviewCompatibleImage imageInfo={project.project_image} /></Link>
+                    <h3 className="has-text-weight-bold is-size-5">
+                      <Link style={{display:'inline-block', color: '#1C2833'}} to={`/Project_${project.id}`}>{project.project_name}</Link>
```
Comment (chris-hammond) [inline src/templates/specific-profile.js]: `has-text-primary`?

File: src/templates/specific-profile.js [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+                    </h3>
+                    <Link style={{color: '#1C2833'}} to={`/Project_${project.id}`}><LinesEllipsis
```
Comment (chris-hammond) [inline src/templates/specific-profile.js]: `has-text-primary`?

**Comment by kjh61998** [comment]
Okay. I believe I got everything changed if y'all wanna take another look :) I also tried to find other spots where we needed "has-text-primary" and fixed a typo on the about page.

File: src/components/all.sass [hunk 7]
```diff
@@ -53,5 +54,26 @@ $body-color: $tech-connect-blue
...
+  display: inline-block
+  list-style: none
+  margin-bottom: -1px
+  border-top-right-radius: 5px
+  border-top-left-radius: 5px
+
+
+.tab-list-active
+  background-color: white
+  border: solid #ccc
```
Comment (chris-hammond) [inline src/components/all.sass]: #ccc should be $tab-border

**Comment by kjh61998** [comment]
> One more and we're good!

Okay now we should be good! Let me know if you see anything else needing fixed :)
